.gitpod.yml

```diff
@@ -2,11 +2,10 @@ tasks:
   - name: Preload
     before: |
       rustup update stable
-      cargo +stable install irust
-      rustup toolchain add nightly-2022-04-06
-      rustup component add rustc-dev --toolchain=nightly-2022-04-06
-      cargo +nightly-2022-04-06 install racer
+      cargo install --path crates/irust
       cargo install cargo-edit cargo-show-asm cargo-expand
+      rustup component add rust-analyzer
+      echo 'export PATH="$PATH:$(rustc --print sysroot)/bin"' >> ~/.bashrc
```

Comment:
This wouldn't work on windows, + from rustup 1.26 (released today) there is no need to manually add rust-analyzer to path.
They added proxy app that will be placed in .cargo/bin which is already in path, and it will be pointed to selected rustup app.
https://github.com/rust-lang/rustup/blob/stable/CHANGELOG.md

So I suggest dependency on rustup 1.26 should be added to invoke `rustup self update`. And after that 
`rustup component add rust-analyzer` should work fine